executor/executor_linux.h

```diff
@@ -103,14 +103,32 @@ static void cover_unprotect(cover_t* cov)
 
 static void cover_mmap(cover_t* cov)
 {
+	char* mapped;
```

Comment:
We don't use C89 start-of-block declarations, please combined declaration with initialization.

Reply:
Done